ci/run.sh

```diff
@@ -59,8 +59,9 @@ clone() {
 # build dmd (incl. building and running the unittests), druntime, phobos
 build() {
     source ~/dlang/*/activate # activate host compiler, incl. setting `DMD`
-    make -j$N -C compiler/src -f posix.mak MODEL=$MODEL HOST_DMD=$DMD DFLAGS="$CI_DFLAGS" BUILD=debug ENABLE_WARNINGS=1 unittest
-    make -j$N -C compiler/src -f posix.mak MODEL=$MODEL HOST_DMD=$DMD DFLAGS="$CI_DFLAGS" ENABLE_RELEASE=1 ENABLE_WARNINGS=1 all
+    $DMD compiler/src/build.d -ofgenerated/build
+    generated/build -j$N MODEL=$MODEL HOST_DMD=$DMD DFLAGS="$CI_DFLAGS" BUILD=debug unittest
```

Comment:
The unittests fail to build with older host compilers, apparently due to `debug` blocks violating `@safe` etc. The original cmdline from the Makefiles (derived from a CI job) looks like this:
```
../../generated/build OS="linux" BUILD="debug" MODEL="64" HOST_DMD="dmd" CXX="c++" AUTO_BOOTSTRAP="" DOCDIR="" STDDOC="" DOC_OUTPUT_DIR="" MAKE="make" VERBOSE="" ENABLE_RELEASE="" ENABLE_DEBUG="" ENABLE_ASSERTS="" ENABLE_LTO="" ENABLE_UNITTEST="" ENABLE_PROFILE="" ENABLE_COVERAGE="" DFLAGS=""  unittest
```

So looks like the rather harmless-looking `ENABLE_DEBUG=""` here has been disabling all `debug` blocks for CI so far; it defaults to `1` with `BUILD=debug`.